vnext/Microsoft.ReactNative.Cxx/JSI/JsiAbiApi.h

```diff
@@ -134,11 +134,11 @@ struct JsiAbiRuntime : facebook::jsi::Runtime {
   bool hasProperty(const facebook::jsi::Object &obj, const facebook::jsi::PropNameID &name) override;
   bool hasProperty(const facebook::jsi::Object &obj, const facebook::jsi::String &name) override;
   void setPropertyValue(
-      facebook::jsi::Object &obj,
+      const facebook::jsi::Object &obj,
```

Comment:
@vmoroz, @chiaramooney Will making these changes cause problems with the ABI surface/hermes?